src/cfnlint/data/AdditionalSpecs/AllowedValues.json

```diff
@@ -0,0 +1,134 @@
+{
+  "ResourceTypes": {
+    "AWS::AmazonMQ::Broker": {
+      "Properties": {
+        "EngineVersion": {
+          "AllowedValues": [
+            "5.15.0"
+          ]
+        },
+        "DeploymentMode": {
+          "AllowedValues": [
+            "SINGLE_INSTANCE",
+            "ACTIVE_STANDBY_MULTI_AZ"
+          ]
+        },
+        "EngineType": {
+          "AllowedValues": [
+            "ACTIVEMQ"
+          ]
+        }
+      }
+    },
+    "AWS::AmazonMQ::Configuration": {
+      "Properties": {
+        "EngineVersion": {
+          "AllowedValues": [
+            "5.15.0"
+          ]
+        },
+        "EngineType": {
+          "AllowedValues": [
+            "ACTIVEMQ"
+          ]
+        }
+      }
+    },
+    "AWS::EC2::VPC": {
+      "Properties": {
+        "InstanceTenancy": {
+          "AllowedValues": [
+            "dedicated",
+            "default"
+          ]
+        }
+      }
+    },
+    "AWS::Lambda::Function": {
+      "Properties": {
+        "Runtime": {
+          "AllowedValues": [
+            "nodejs",
+            "nodejs4.3",
+            "nodejs6.10",
+            "nodejs8.10",
+            "java8",
+            "python2.7",
+            "python3.6",
+            "dotnetcore1.0",
+            "dotnetcore2.0",
+            "dotnetcore2.1",
+            "nodejs4.3-edge",
+            "go1.x"
+          ]
+        }
+      }
+    },
+    "AWS::RDS::DBCluster": {
+      "Properties": {
+        "Engine": {
+          "AllowedValues": [
+            "aurora",
+            "aurora-mysql",
+            "aurora-postgresql"
+          ]
+        },
+        "EngineMode": {
+          "AllowedValues": [
+            "provisioned",
+            "serverless"
+          ]
+        }
+      }
+    }
+  },
+  "PropertyTypes": {
+    "AWS::DMS::Endpoint": {
```

Comment:
Shouldn't this have a property with a `.` between?

Reply:
Yeah, wrong place... Will fix